index.js

```diff
@@ -43,7 +43,16 @@ class MarkdownButtonElement extends HTMLElement {
 class MarkdownHeaderButtonElement extends MarkdownButtonElement {
   constructor() {
     super()
-    styles.set(this, {prefix: '### '})
+
+    if (!this.hasAttribute('level')) {
```

Comment:
What do you think about using [data attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) instead of a non-standard attribute here?

Reply:
Makes sense, will update

Reply:
ACTUALLY! We apperantly _do_ use non-standard attributes on custom elements so this is actually what we want 😬 

Sorry about the confusion.